artifacts/openemr/openemr-deployment.yaml

```diff
@@ -59,7 +59,7 @@ spec:
           value: "redis"
         - name: SWARM_MODE
           value: "yes"
-        image: "openemr/openemr:5.0.2"
+        image: "quay.io/bostrt/openemr-test"
```

Comment:
Note that https://quay.io/organization/openemr was created

Reply:
Updated @shawndwells